openedx/core/djangoapps/user_api/accounts/tests/test_api.py

```diff
@@ -509,13 +509,15 @@ class AccountSettingsOnCreationTest(CreateAccountMixin, TestCase):
     PASSWORD = 'ṕáśśẃőŕd'
     EMAIL = 'frank+underwood@example.com'
     ID = -1
+    ACTIVATION_KEY = None
```

Comment:
seems like this is only needed in a single test, why you have defined it at the class level?

Reply:
I have refactored it to directly read from user dict now, won't be needing class level variables now.